higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -92,20 +92,60 @@ <h2 class="heading-secondary">Personal Questionnaire</h2>
                   </ul>
                 </div>
               </div>
+            </div>
+
+            <div class="login-form__medical">
+              <h2 class="heading-secondary">Where are you going today?</h2>
+              <ul id="medical" class="fieldset">
+                {% for field in form %}
+                  {% if "facility_" in field.name %}
+                    {% if field.name == "facility_destination" %}
+                      <h3 class="subheading">Where are you going today?</h3>
+                    {% elif field.name == "facility_destination_university" %}
+                      <h3 class="subheading">Educational institution information</h3>
+                    {% endif %}
```

Comment:
We should definitely have some javascript to filter the `university` dropdown when the `campus` field is supplied. And to similarly filter the `campus` dropdown when you select a `province`, otherwise these dropdowns are going to be massive and not very user friendly.